lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -2625,10 +2633,20 @@ public function testArrayConditionsOneElement() {
 		$expected = " WHERE id = (1)";
 		$this->assertEquals($expected, $result);
 
+		$conditions = array('key' => array('939'));
+		$result = $this->Dbo->conditions($conditions);
+		$expected = " WHERE `key` IN ('939')";
```

Comment:
Do you have any idea in what cases `WHERE <X> = (<Y>)` would make sense (or is optimal)?